keyboards/atreus/keymaps/kejadlen/config.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#undef MATRIX_ROW_PINS
+#undef MATRIX_COL_PINS
+#define MATRIX_ROW_PINS { D0, D1, D2, D3 }
+#define MATRIX_COL_PINS { F6, F5, F4, F1, F0, F7, B0, B1, B2, B3, B7 }
+/* #define UNUSED_PINS */
+
+#define DIODE_DIRECTION COL2ROW
+
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This define doesn't do anything anymore, and the behavior it enabled is now the default for QMK.

```suggestion
```